tests/test_constants.py

```diff
@@ -43,8 +42,8 @@ def setUp(self) -> None:
 
     def test_constants(self):
         """Test types."""
-        self.assertIsInstance(OS_CONFIG, dict)
-        os_config = OS_CONFIG.get(opersys)
+
+        os_config = OSConfiguration.from_opersys(opersys)
```

Comment:
argument `opersys` is already the implicit default value

Reply:
You're right. Removed.